src/api/mocks/src/meshes/_/meshgateways.ts

```diff
@@ -0,0 +1,42 @@
+import type { EndpointDependencies, MockResponder } from '@/api/mocks/index'
+export default ({ fake }: EndpointDependencies): MockResponder => (req) => {
+  const params = req.params
+  const total = fake.datatype.number(200)
+
+  return {
+    headers: {},
+    body: {
+      total,
+      items: Array.from({ length: total }).map((_, i) => {
```

Comment:
**Comment**: Btw, just a note: You can provide the map function as the second argument of `Array.from` without the need to chain off an explicit `map` call.